apps/io-services-cms-webapp/src/webservice/controllers/edit-service.ts

```diff
@@ -27,9 +27,12 @@ import {
   itemToResponse,
   payloadToItem,
 } from "../../utils/converters/service-lifecycle-converters";
+import { IConfig } from "../../config";
 
 type Dependencies = {
   fsmLifecycleClient: ServiceLifecycle.FsmClient;
+  // Configuration
```

Comment:
nitpick: avoid to insert useless comment 😉 
```suggestion
```

Reply:
[0f02588](https://github.com/pagopa/io-services-cms/pull/115/commits/0f02588285efb043be4d3f79578e119405cc6eda)